src/code/z_fbdemo_circle.c

```diff
@@ -25,7 +25,7 @@ Gfx sTransCircleDL[] = {
     gsDPLoadTextureBlock(0x08000000, G_IM_FMT_I, G_IM_SIZ_8b, 16, 64, 0, G_TX_NOMIRROR | G_TX_WRAP,
                          G_TX_NOMIRROR | G_TX_CLAMP, 4, 6, G_TX_NOLOD, G_TX_NOLOD),
     gsSPDisplayList(0x09000000),
-    gsSPVertex(sTransCircleVtx, 32, 0),
+    gsSPVertex(sTransCircleDL, 32, 0),
```

Comment:
This looks sus

Reply:
the formatter did this.....????????

Reply:
or maybe not, i have no idea. very lost lol

Reply:
oh i think its due to running the formatter without up-to-date assets. I have no idea why its changing this though, it should only be changing args in function declarations afaik